changelog.md

```diff
@@ -18,6 +18,7 @@
  ### WML Engine
    * Modify implementation of overwrite_specials attribute for replace yes/no parameter by none/one_side/both_sides and select abilities used like weapons and specials who must be overwrited(owned by fighter where special applied or both)
    * Add a 'ability_id_active' attribute to [filter]
+   * Add [immune] abilities who have similar effect what undrainable or unpoisonable status, and aplied also on slow plague or petrifies.
```

Comment:
Typo: "applied" has two Ps. Also, use "which" instead of "who" and replace "what" with either "to" or "as", and a comma after "slow".